src/libOpenImageIO/icc.cpp

```diff
@@ -131,20 +137,32 @@ icc_color_space_name(string_view color_space)
         { "BCLR", "12 color" }, { "CCLR", "13 color" }, { "DCLR", "14 color" },
         { "ECLR", "15 color" }, { "FCLR", "16 color" },
     };
-    return color_space_names[color_space];
+    // std::unordered_map::operator[](const Key& key) will add the key to the map if
+    // it doesn't exist. This isn't what is intended and isn't thread safe.
+    // Instead, just do the lookup and return the value or a nullptr.
+    //
+    // return color_space_names[color_space];
+    auto it = color_space_names.find(color_space);
+    return (it != color_space_names.end()) ? it->second : nullptr;
+
 }
 
 static const char*
 icc_primary_platform_name(const std::string& platform)
 {
-    static std::unordered_map<std::string, const char*> primary_platforms = {
+    static const std::unordered_map<std::string, const char*> primary_platforms = {
         { "APPL", "Apple Computer, Inc." },
         { "MSFT", "Microsoft Corporation" },
         { "SGI ", "Silicon Graphics, Inc." },
         { "SUNW", "Sun Microsystems, Inc." },
     };
-    return primary_platforms[platform];
-}
+    // std::unordered_map::operator[](const Key& key) will add the key to the map if
+    // it doesn't exist. This isn't what is intended and isn't thread safe.
+    // Instead, just do the lookup and return the value or a nullptr.
+    //
+    // return primary_platforms[platform];
+    auto it = primary_platforms.find(platform);
+    return (it != primary_platforms.end()) ? it->second : nullptr;
```

Comment:
I think you lost a `}` at this line...

Reply:
Sorry for the carelessness. I added back the missing `}`